spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe "Mechanics Show Page" do
+  before(:each) do
+    @amusement_park = AmusementPark.create!(name: 'Six Flags', admission_cost: 75)
+    @mechanic1 = Mechanic.create!(name: "Larry", years_experience: 3)
+    @mechanic2 = Mechanic.create!(name: "Ben", years_experience: 10)
+
+    @ride1 = @mechanic1.rides.create!(name: "The Hurler", thrill_rating: 7, open: false, amusement_park_id: @amusement_park.id)
+    @ride2 = @mechanic1.rides.create!(name: "Dragon Hunter", thrill_rating: 6, open: false, amusement_park_id: @amusement_park.id)
+    @ride3 = @mechanic2.rides.create!(name: "Twister", thrill_rating: 9, open: false, amusement_park_id: @amusement_park.id)
+    @ride4 = @mechanic2.rides.create!(name: "Merchant of Death", thrill_rating: 10, open: false, amusement_park_id: @amusement_park.id)
+    @ride5 = Ride.create!(name: "Fun Time", thrill_rating: 2, open: false, amusement_park_id: @amusement_park.id)
+  end
+
+  describe 'User Story 1' do
+    it "Displays a mechanics attributes" do
+      visit mechanic_path(@mechanic1)
+
+      expect(page).to have_content(@mechanic1.name)
+      expect(page).to have_content(@mechanic1.years_experience)
+
+      expect(page).to_not have_content(@mechanic2.name)
+      expect(page).to_not have_content(@mechanic2.years_experience)
+    end
```

Comment:
I feel like I've made this comment several times about these types of tests. When we look at line 21, what is our assertion? `expect(page).to have_content(@mechanic1.years_experience)` is the same as `expect(page).to have_content(3)` right? Now just looking at that assertion, does it make a whole lot of sense to just be looking for `3` on the page? Probably not. This assertion has the potential to create false positives and would not be considered a good test.

Here are two better assertions for numbers:

```ruby
within "#mechanic_#{@mechanic1.id}" do
  expect(page).to have_content(@mechanic1.years_experience)
end
```

or 

```ruby
expect(page).to have_content("Years of experience: #{@mechanic1.years_experience}")
```

Regardless of your choice, it would be best to make your assertions clear so that they don't mistakenly result in a false positive, especially when it comes to number values.